src/loc.rs

```diff
@@ -56,10 +56,27 @@ impl Loc {
     }
 }
 
+/// The set of errors which may occur.
+///
+/// This enum describes the possible errors when parsing a loc.  A valid loc
+/// consists of
+/// * the integral number of insertions,
+/// * the integral number of deletions, and
```

Comment:
This explains the hyphen in the parsing function therefore.  As of my Git version, `0` instead of `-` is written then.  I will add this.

Reply:
What is your git version? Mine is `2.34.0.windows.1` and for binary files it outputs - instead of 0, because 0 means renaming or something like that I think 😅

Reply:
`2.30.2`

Reply:
The LOC diff for `1ab5121` looks as follows with `git log --numstat`:

```
10      0       src/lib.rs
```

Reply:
:wrench: :heavy_check_mark:

Reply:
> The LOC diff for `1ab5121` looks as follows with `git log --numstat`:
> 
> ```
> 10      0       src/lib.rs
> ```

Yeah, but the '-' is for binary files, like images, videos or other things, not for source files.